pkg/server/status.go

```diff
@@ -2889,13 +2889,23 @@ func (s *systemStatusServer) HotRangesV2(
 		ErrorsByNodeID: make(map[roachpb.NodeID]string),
 	}
 
-	var requestedNodes []roachpb.NodeID
-	if len(req.NodeID) > 0 {
-		requestedNodeID, local, err := s.parseNodeID(req.NodeID)
+	nodes := req.Nodes
+	if req.NodeID != "" {
+		nodes = append(nodes, req.NodeID)
```

Comment:
nit: no need to change, but in the future, I don't think we should mix support for fields like `Nodes` and `NodeID`. I'd rather have one take precedence to reduce possible combinations.